aayushisingh11/CountZeros.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+int CountZeros(int arr[], int low, int high)
```

Comment:
it's not guaranteed that the array will be sorted, so a binary search is not appropriate here. How about just counting from beginning to the end?

Reply:
Oh, sorry I did mistake. I'll try to resolve it as you suggested. Thank you!

Reply:
Oh, sorry I did mistake. I'll try to resolve it as you suggested. Thank you!